includes/Traits/Assets.php

```diff
@@ -84,32 +105,57 @@ public function register_script( $script_handle, array $script_dependencies = []
 	 * @param string $script_handle Handle of script.
 	 * @param array  $script_dependencies Array of extra dependencies.
 	 *
-	 * @return void
+	 * @return array Array containing $script_handle and all dependent script chunk handles.
 	 */
 	public function enqueue_script( $script_handle, array $script_dependencies = [] ) {
-		$this->register_script( $script_handle, $script_dependencies );
+		$asset = $this->register_script( $script_handle, $script_dependencies );
+
+		// Enqueue chunks of $script_handle.
+		$script_chunks = $asset['js'];
+		foreach ( $script_chunks as $chunk ) {
+			wp_enqueue_script( $chunk );
+		}
```

Comment:
This is redundant and not needed.

If you enqueue a script, all its dependencies will be enqueued as well.

And since you already added all chunks to `$script_dependencies` in `register_script()` above, they will be loaded as expected.

Reply:
I thought I tried that but I'll check again. :)